scico/examples.py

```diff
@@ -287,12 +287,59 @@ def create_circular_phantom(
 
     img = snp.zeros(img_shape)
     for r, val in zip(radius_list, val_list):
-        # img[dist_map < r] = val
+        # img[dist_map < r] = val in numpy
         img = img.at[dist_map < r].set(val)
 
     return img
 
 
+def create_3D_foam_phantom(
+    im_shape: Shape,
+    N_sphere: int,
+    r_mean: float = 0.1,
+    r_std: float = 0.001,
+    pad: float = 0.01,
+    is_random: bool = False,
+):
+    """Construct a 3D phantom with random radii and centers.
+
+    Args:
+        im_shape: Shape of input image.
+        N_sphere: Number of spheres added.
+        r_mean: Mean radius of sphere (normalized to 1 along each axis).
+                Default 0.1.
+        r_std: Standard deviation of radius of sphere (normalized to 1
+                along each axis). Default 0.001.
+        pad: Padding length (normalized to 1 along each axis). Default 0.01.
+        is_random: Flag used to controll randomness of phantom generation.
+                If `False`, random seed is set to 1 in order to make the
```

Comment:
Should be double backticks on ``False``.

Reply:
Fixed it.